<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zdyBrandDig</class>
 <widget class="QDialog" name="zdyBrandDig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>353</width>
     <height>601</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imagesRC.qrc">:/images/zdyIcon/zdyBackground.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="zdyVerBtn">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>405</y>
     <width>171</width>
     <height>187</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imagesRC.qrc">
     <normaloff>:/images/zdyIcon/10000-1.png</normaloff>
     <activeoff>:/images/zdyIcon/10000-2.png</activeoff>:/images/zdyIcon/10000-1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>171</width>
     <height>187</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="zdyCenterBtn">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>205</y>
     <width>171</width>
     <height>187</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imagesRC.qrc">
     <normaloff>:/images/zdyIcon/1000-1.png</normaloff>
     <activeoff>:/images/zdyIcon/1000-2.png</activeoff>:/images/zdyIcon/1000-1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>171</width>
     <height>187</height>
    </size>
   </property>
  </widget>
  <widget class="QTextEdit" name="zdyTextEdit_v">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>470</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:600;&quot;&gt;中奖&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:600;&quot;&gt;10,000元&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif'; font-size:9pt; font-weight:600;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enterBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>550</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zdyBottomBtn">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>171</width>
     <height>187</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imagesRC.qrc">
     <normaloff>:/images/zdyIcon/500-1.png</normaloff>
     <activeoff>:/images/zdyIcon/500w-2.png</activeoff>
     <selectedoff>:/images/zdyIcon/500w-2.png</selectedoff>:/images/zdyIcon/500-1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>171</width>
     <height>187</height>
    </size>
   </property>
  </widget>
  <widget class="QTextEdit" name="zdyTextEdit_ct">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-weight:600;&quot;&gt;中奖&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-weight:600;&quot;&gt;1,000元&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="zdyTextEdit_botm">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-weight:600;&quot;&gt;中奖5,000,000&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-weight:600;&quot;&gt;元0角0分&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>zdyBottomBtn</tabstop>
  <tabstop>zdyCenterBtn</tabstop>
  <tabstop>zdyVerBtn</tabstop>
  <tabstop>zdyTextEdit_botm</tabstop>
  <tabstop>zdyTextEdit_ct</tabstop>
  <tabstop>zdyTextEdit_v</tabstop>
  <tabstop>enterBtn</tabstop>
 </tabstops>
 <resources>
  <include location="imagesRC.qrc"/>
 </resources>
 <connections/>
</ui>
